app/controllers/api/v1/search_controller.rb

```diff
@@ -3,7 +3,7 @@ def index
     trips = TripFacade.get_trips(flight_params)
     if trips.class == Hash
       payload = {
-          error: "Invalid Data",
+          error: trips[:message].first[:errors].first,
```

Comment:
Nice! I like this

Reply:
I like the use of specific error messages returned by Kiwi here!